albumentations/augmentations/geometric/functional.py

```diff
@@ -377,22 +379,6 @@ def elastic_transform(
     return remap_fn(img)
 
 
-@preserve_channel_dim
-def resize(img, height, width, interpolation=cv2.INTER_LINEAR):
```

Comment:
@Dipet Is it an intended change to move `resize` and `scale` from `albumentations/augmentations/geometric/functional.py` to `albumentations/augmentations/functional.py`?

Reply:
Yes, this is intentional because I don't know how to import them from `geometric.functional` to avoid infinite imports (`augmentations.geometric.functional` imports some names from `augmentations.functional`)